code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -461,7 +464,9 @@
 	SSair.add_to_rebuild_queue(src)
 
 /obj/machinery/atmospherics/update_name()
-	name = "[GLOB.pipe_color_name[pipe_color]] [initial(name)]"
+	if(!name_initialised)
+		name = "[GLOB.pipe_color_name[pipe_color]] [name]"
+		name_initialised = TRUE
```

Comment:
IIRC changing a var from default value does make the object use more memory, so doing this for EVERY pipe in the game on the off chance you _might_ have one that is vareditted is the reverse of how this should be handled.

Reply:
also this doesn't fix scrubbers / vents so you are aware.

Reply:
Yeah just looked at vents and scrubbers and those needs more changes